### ISSUE TITLE
Respect svelte-ignore pragma comments for parity with official svelte-check

### ISSUE THREAD

**Issue by ericsampson** [pheuter/svelte-check-rs#20]
## Summary

`svelte-check-rs` does not respect `<!-- svelte-ignore -->` pragma comments, causing warnings to be emitted even when they've been explicitly suppressed. The official `svelte-check` correctly respects these pragmas.

## Current Behavior

When a Svelte file contains a pragma comment like:

```svelte
<!-- svelte-ignore a11y_no_noninteractive_tabindex -->
<div tabindex="0">Interactive content</div>
```

`svelte-check-rs` still emits the warning:
```
Warning: A11y: Non-interactive elements like <div> should not have tabindex (a11y-no-noninteractive-tabindex)
```

## Expected Behavior

The warning should be suppressed, matching the behavior of the official `svelte-check` which correctly respects `svelte-ignore` pragmas.

## Reproduction

A minimal reproduction is available at: https://github.com/ericsampson/svelte-check-rs-bug-repros

```bash
git clone https://github.com/ericsampson/svelte-check-rs-bug-repros
cd svelte-check-rs-bug-repros
bun install
bun run repro
```

The test file `src/routes/pragma-test.svelte` contains:
```svelte
<!-- svelte-ignore a11y_no_noninteractive_tabindex -->
<div tabindex="0">has pragma - should NOT warn</div>
```

**svelte-check (official)**: 0 warnings for this file ✅
**svelte-check-rs**: 1 warning for this file ❌

## Implementation Notes

The pragma syntax supports multiple forms:
- `<!-- svelte-ignore a11y_no_noninteractive_tabindex -->` (specific rule)
- `<!-- svelte-ignore a11y-no-noninteractive-tabindex -->` (hyphenated variant)
- Multiple rules: `<!-- svelte-ignore a11y_rule1 a11y_rule2 -->`

The pragma should apply to the immediately following element/block.

## Related

- #19 (tsconfig exclude not respected)
- Official svelte-check pragma documentation: https://svelte.dev/docs/svelte/svelte-ignore
